research/picai/prepare_data.py

```diff
@@ -0,0 +1,209 @@
+import argparse
+import json
+import os
+from pathlib import Path
+from typing import Optional, Sequence, Tuple
+
+import numpy as np
+import SimpleITK as sitk
+from preprocessing import (
```

Comment:
Is this from the PICAI preprocessing library? It seemed like maybe we removed it in the pyproject.toml